### DISCUSSION TITLE
Push notifications for releases

### DISCUSSION THREAD

**Discussion by jeankhawand** [community/community#141481] [Mobile] [answered]
I am currently monitoring release events on a specific repository, and while these events appear in app notification tab, I am not receiving push notifications to alert me about them.

**Comment by stevepopovich** [comment]
Hi @jeankhawand at this time, we do not support push notifications for releases. You can see what push notifications we send here:
![Screenshot_20241011_121731](https://github.com/user-attachments/assets/aa007a39-740c-4a6d-9695-2b134900b0b9)

Profile -> Settings -> Notification Options

**Reply by jeankhawand** [reply]
Thanks for your response, is there a way to raise feedback item to consider it in future releases

**Reply by stevepopovich** [reply, marked as answer]
Oh yes. I have already filed it as a feature request, and we will consider it. Thank you!
